lib/src/components/NxSmallTag/NxSmallTag.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import { Props, propTypes } from './types';
+
+import './NxSmallTag.scss';
+export { Props } from './types';
+
+const NxSmallTag = forwardRef<HTMLLabelElement, Props>(function NxSmallTag(props, ref) {
+  const { children, className, color, ...attrs } = props,
+      tagClasses = classnames('nx-small-tag', className, {
+        'nx-small-tag-color--pink': !color,
+        [`nx-small-tag-color--${color}`]: color
+      });
+
+  return (
+    <label className={tagClasses} ref={ref} {...attrs}>
+      <span>{children}</span>
```

Comment:
What's the purpose of the span?

Reply:
No specific purpose, removed here 30e0f308bfe4d3c84213449914d1df186215aa2b.